src/components/Hero/_Hero.scss

```diff
@@ -165,3 +165,16 @@
     }
   }
 }
+
+.hero--tertiary {
+  background-color: var(--ui-gray-xdark);
```

Comment:
Are the background-color variants being picked up by the classes in `.hero--secondary`?

Reply:
The color variants will be added through the `backgroundColor` parameter, same as the Hero Primary or Hero 5050 would pass the background color.  
I suppose background color can be added the same way as Hero Secondary, but I was running into the lint issue described in the previous comment.

Reply:
OK, that works.